WordPress/Classes/ViewRelated/Me/AppSettingsViewController.swift

```diff
@@ -195,6 +194,11 @@ class AppSettingsViewController: UITableViewController {
         return section == Sections.editor.rawValue && EditorSettings().editor == .aztec
     }
 
+    func handleEditorFooterTap(_ sender: UITapGestureRecognizer) {
```

Comment:
Let's make this `private` – but you'll also need to add `@objc` so the gesture recogniser can find it.

`@objc private func handleEditorFooterTap(...`